ci-operator/step-registry/ipi/install/libvirt/install/ipi-install-libvirt-install-ref.yaml

```diff
@@ -6,6 +6,7 @@ ref:
   env:
   - name: ARCH
   - name: BRANCH
+  - name: INSTALLER
```

Comment:
I think we need to set a default, otherwise all of our job configs will need to set a value.

It would be good to add a documentation block as well:

```
  - name: INSTALLER
    default: "libvirt"
    documentation: |-
      The installation target for the job. Valid values are ['libvirt', 'powervs']
````